resolwe/flow/execution_engines/bash/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""An execution engine that outputs bash programs."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from django.conf import settings
+
+from resolwe.flow.execution_engines import BaseExecutionEngine, ExecutionError
```

Comment:
Why would you import it there? The execution engines need to be defined in the settings file and are registered by the manager.